components/ServerStatus.jsx

```
@@ -0,0 +1,53 @@
+        setLoading(false);
+      }
+    };
+
+    fetchData();
+
+    const pollInterval = setInterval(fetchData, 30000); // Poll every 30 seconds
+    return () => clearInterval(pollInterval);
+  }, []);
+
+  return (
+    <div className="flex items-center" role="status" aria-live="polite">
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<output>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)